presave.js

```diff
@@ -0,0 +1,18 @@
+var H5PPresave = H5PPresave || {};
+
+H5PPresave['H5P.MarkTheWords'] = function (content, finished) {
+    var score = 0;
+    if( content.hasOwnProperty('textField') === true ){
+        var pattern = /\*[^\*\s]+\*/g;
+        score = content.textField.match(pattern || []).length;
```

Comment:
This does not account for multi-word answers that users may have made, e.g. *multi word answer*. 
It also does not account for words that has escaped asterisks within them, e.g.: *cor**rect*

Reply:
The max score logic should optimally be exactly the same as the score parser that is used within the view.

Reply:
Ok. In the Important instructions in the editor it says that only words may be marked as correct, not phrases. That's why I didn't add support for multiple word answers.
I'll fix it.